src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -97,14 +97,18 @@ public Token login(String username, String password)
      * @return authentication token as {@link Token}
      * @throws SaltStackException if anything goes wrong
      */
-    public Token login(String username, String password, String eauth)
+    public Token login(final String username, final String password, final String eauth)
             throws SaltStackException {
-        JsonObject json = new JsonObject();
-        json.addProperty("username", username);
-        json.addProperty("password", password);
-        json.addProperty("eauth", eauth);
+        Map<String, String> kwargs = new LinkedHashMap() {
```

Comment:
While I am a fan of this style of inline initialization of `Map`s and `List`s, I am not a big fan of `final` method parameters, [as pointed out in another PR](https://github.com/SUSE/saltstack-netapi-client-java/pull/28#commitcomment-9865167). Maybe it is just my personal taste, though. @renner, your idea?